criu/plugin.c

```diff
@@ -243,7 +243,11 @@ int cr_plugin_init(int stage)
 		if (len < 3 || strncmp(de->d_name + len - 3, ".so", 3))
 			continue;
 
-		snprintf(path, sizeof(path), "%s/%s", opts.libdir, de->d_name);
+		if (snprintf(path, sizeof(path), "%s/%s", opts.libdir, de->d_name) >=
+		    sizeof(path)) {
```

Comment:
Instead of `sizeof(path)` wouldn't it be simpler to just use `PATH_MAX`? In `snprintf()` and in the compare?

Reply:
I actually like sizeof() option more as a more universal way, and as less error prone. I can do better and add size variable so that we don't need to calculate sizeof twice. I thought prefered to make diff smaller. -> will add size variable if you like.

Reply:
Not necessary from my point of view.